Tensile/Common.py

```diff
@@ -310,7 +310,16 @@
   'gfx1100':'navi31', 'gfx1101':'navi32', 'gfx1102':'navi33'
 }
 
-def getArchitectureName(gfxName):
+def getArchitectureName(gfxName: str) -> Optional[str]:
+  """Maps the provided Gfx architecture to its common name using the **architectureMap**.
+
```

Comment:
```suggestion
  """Maps the provided Gfx architecture to its common name using the **architectureMap**. If gfxName is is an exact match or a substring of a key in **architectureMap**, the function will return the corresponding value.

```

Reply:
Not applying as this info is already in the return part of the doc-string